ompi/mca/topo/base/base.h

```diff
@@ -194,6 +194,9 @@ OMPI_DECLSPEC int
 mca_topo_base_dist_graph_neighbors_count(ompi_communicator_t *comm,
                                          int *inneighbors, int *outneighbors, int *weighted);
 
+
```

Comment:
``ompi_comm_neighbors_count`` using ``MPI_*`` interfaces interfere with MPI profiling. I'm in favor of the new routine. Moreover, the new routine should be used everywhere and the former should be removed.